### DISCUSSION TITLE
Reference to network planning for Data Management Zone deployment

### DISCUSSION THREAD

**Discussion by renepajta** [Azure/data-management-zone#113] [General] [answered]
I would appreciate a link (Learn more) for network planning to better understand what is gonna be deployed and better estimate how large address ranges I need

![image](https://user-images.githubusercontent.com/6043628/127496131-5cbc5a92-e474-4ab4-a824-1142b48463df.png)

**Comment by marvinbuss** [comment, marked as answer]
@renepajta Thanks for the suggestion. The link is a really good idea. Have you seen that we have checks on the CIDR range? If the user chooses a too small address space, the field should show an error. Is it not showing that behavior for you?

@mboswell do we have a doc about network planning in CAF to which I can link here?

**Reply by mboswell** [reply]
@renepajta and @marvinbuss we don't have anything in the CAF today but it would be good to calls this out in the planning section. 

We will put it into our backlog.

**Comment by renepajta** [comment]
Thank you!
